app.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express");
+const app = express();
+
+const { getTopics } = require("./controllers/topics-controllers");
+
+app.use(express.json());
```

Comment:
You aren't handling a body yet, this does not need to be here.

Reply:
I'm still a bit shaky on what this does so I popped it in just in case, but I understand that is not good practice. I'll remove it, read up and re enter when necessary